definitions/npm/node-cache_v4.x.x/flow_v0.25.x-/node-cache_v4.x.x.js

```diff
@@ -0,0 +1,24 @@
+// @flow strict
```

Comment:
Although I don't think it makes any harm, `// @flow strict` is not actually needed in libdefs.

Reply:
Cheers, good to know 👍 — fixed in 572f99a